src/inputs/internal/SelectFieldBase.tsx

```diff
@@ -72,6 +73,7 @@ export function SelectFieldBase<O, V extends Value>(props: BeamSelectFieldBasePr
     unsetLabel,
     ...otherProps
   } = props;
+  const labelStyle = fieldProps?.labelStyle ?? "above";
```

Comment:
Thought: Do we need to check `otherProps.labelStyle`?

Reply:
oh yea,... we probably should... 👍